packages/cashier/src/pages/payment-agent/payment-agent-withdraw-form/__tests__/payment-agent-withdraw-form.spec.tsx

```diff
@@ -42,11 +42,13 @@ describe('<PaymentAgentWithdrawForm />', () => {
     });
 
```

Comment:
@hamid-deriv Is there any reason for changing test files to `tsx` while the test folder is ignored in the typescript config file? 🤔